go/host/host.go

```diff
@@ -984,7 +987,7 @@ func (h *host) startBatchStreaming() {
 				lastBatch = resp.Batch
 				h.logger.Trace("Received batch from stream", log.BatchHashKey, lastBatch.Hash())
 				if h.config.NodeType == common.Sequencer {
-					h.logger.Info("Storing batch from stream")
+					h.logger.Info("Batch produced", "height", resp.Batch.Header.Number)
```

Comment:
Be good to log the hash as well here, and again you could use the constants for the keys. (Datadog charges by log event rather than by data size so no need to be stingy with the metadata)